### DISCUSSION TITLE
Sound quality Deutsch not the best.

### DISCUSSION THREAD

**Discussion by manojubhatia** [coqui-ai/TTS#1421] [General]
Hi, I am warming up to this repo. Had the same issue here, and After installing espeak, the problem is gone. But the voice that gets loaded has a clinging sound. Am I doing something wrong, or is this the quality of the model? The voice does match @thorstenMueller though, but the quality is not very appealing. Is it my setup? That clinging sound is more pronounced when the playback speed is slower. Is this the filler noise?
Also when I submit a sentence without a punctuation at the end, the sound file instead of 2 seconds goes on for 23secs.
Have to still try the other voice models.

_Originally posted by @manojubhatia in https://github.com/coqui-ai/TTS/issues/1408#issuecomment-1069587772_

**Comment by thorstenMueller** [comment]
When you share a sample i can check if it's the models quality or an issue with your setup.
Just in case. A new model / dataset in work-in-progress and it has a more natural speechflow and a better quality. You can check the samples on website of [Thorsten-Voice](https://www.thorsten-voice.de/en/2022/03/07/audio-samples-of-next-thorsten-voice-tts-model/) project.

**Reply by manojubhatia** [reply]
I have found and followed your instructions from the Youtube video "Setup Coqui TTS server in less than 5 minutes".  Would you like to have a sample of the voice I get back from this setup?
